### PR TITLE
Fix for a typo in runregr_udr, causing udr/TEST002 failure

### PR DESCRIPTION
**Pull request by zellerh**
The problem was reported by Eric over the weekend and Arvind found
the bug and suggested this fix.

The failure occurs when a user installed Hadoop using non-standard ports and if there is no running instance on the default ports.

### CODE REVIEW

**Comment by DaveBirdsall** [comment]
Looks good to me.

File: core/sql/regress/tools/runregr_udr.ksh [hunk 1]
```diff
@@ -851,9 +851,9 @@ if [ $LINUX -ne 0 ]; then
 fi
```
Comment (hegdean) [inline core/sql/regress/tools/runregr_udr.ksh]: Do we want to clean up the unused  env variables/conditional checks ( _RLD_LIB_PATH and CLASSPATH) ?

Comment (zellerh) [inline core/sql/regress/tools/runregr_udr.ksh]: There is a lot of cleanup that could be done here, but that would require more testing and take more time. Is it ok if we keep this PR to just the typo?

Comment (hegdean) [inline core/sql/regress/tools/runregr_udr.ksh]: Should be fine Hans.  Just did not want to see hpt4jdbc.jar being referenced anywhere incorrectly as our tyep4 jar file is now called jdbcT4.jar

Comment (zellerh) [inline core/sql/regress/tools/runregr_udr.ksh]: Thanks, now I understand why you are suggesting this. I'll push an update shortly.

**Comment by arvind-narain** [comment]
Looks good Hans - thanks. Merging in now.

**Pull request merged**